tests/Rector/ClassMethod/MakeEntitySetterNullabilityInSyncWithPropertyRector/Fixture/fixture.php.inc

```diff
@@ -34,13 +56,35 @@ use Doctrine\ORM\Mapping as ORM;
 class Product
 {
     /**
-     * @ORM\ManyToOne(targetEntity="AnotherEntity")
+     * @ORM\ManyToOne(targetEntity="DefaultEntity")
      */
-    private $anotherEntity;
+    private $defaultEntity;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="NotNullableEntity")
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private $notNullableEntity;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="NullableEntity")
+     * @ORM\JoinColumn(nullable=true)
+     */
+    private $nullableEntity;
+
+    public function setDefaultEntity(?DefaultEntity $defaultEntity)
+    {
+        $this->defaultEntity = $defaultEntity;
+    }
+
+    public function setNotNullableEntity(NotNullableEntity $notNullableEntity)
+    {
+        $this->notNullableEntity = $notNullableEntity;
+    }
 
-    public function setAnotherEntity(AnotherEntity $anotherEntity)
+    public function setNullableEntity(?NullableEntity $nullableEntity)
     {
-        $this->anotherEntity = $anotherEntity;
+        $this->nullableEntity = $nullableEntity;
```

Comment:
Could you add this as separate fixture? So we have clear separated fixtures per feature. It is much easier to git blame and git history on separated files :)

Reply:
created foreach case an own test files as there are 3 cases (not set, nullable true, nullable false) I creatd 6 test cases to test all cases in combination with the setter.